portfolio/src/main/webapp/script.js

```diff
@@ -475,6 +475,10 @@ function formatCommentReply(comment) {
  * isUpvote is false. 
  */
 function changeVote(comment, isUpvote, amount) {
+  // Prevent a POST request from changing vote count by more than 1
+  if(amount !== 1 && amount !== -1) {
```

Comment:
space after `if`

and next line should be 2-space indentation

Cloud shell should be able to format JS file correctly?

Reply:
Sorry about that, I forgot to format the JS file. Cloud shell does automatically format JS files, but it keeps resetting the indentation to 4 space.

Reply:
You should be able to set the indentation to 2-space in cloud shell 
https://photos.app.goo.gl/qutP9pwXC9waV2a96